README.md

```diff
@@ -1,90 +1,103 @@
+```
 
-_NOTE:_ The first time you build it will take a while (~45minutes). After that Docker cache will speed it up.
+```
+# Using python 3
+$ export DM_PACKAGES_URL=http://localhost:9999
+$ cd local_sdks && python -m http.server 9999
+```
 
-### Start
-* Then, start a container based on that image by running: `./server/scripts/run-local.sh`.
-* The build server is now available on port `http://localhost:9000`
+### Build the Docker image
+Build the Extender Docker image by running:
 
-### Stop
-* Just hit `Ctrl-C`.
+```
+./server/scripts/build.sh
+```
 
-### Standalone server
+To speed things up, tests can be disabled by passing `-xtest` to the command line.
 
-The stand alone server is currently used on a machine runing macOS.
-The server is used to build darwin targets (macOS+iOS) using the Apple tools (XCode+Apple Clang)
+```
+$  ./server/scripts/build.sh -xtest
+```
 
-#### Run
-To run the stand alone server locally, you need to give it access to `/usr/local/extender`:
+NOTE: The first time you build it will take a while (approx. 45 minutes). After that the Docker cache will speed it up.
 
-        $ sudo mkdir /usr/local/extender
-        $ sudo chown -R mawe:staff /usr/local/extender
 
-Now the current user has access to the folder and can start the service.
+### Run the Docker image in a container
+Start the container based on the Docker image that was built by running:
 
-        $ TARGET_DIR=/path/to/localextender ./server/scripts/run-standalone-local.sh
+```
+$  ./server/scripts/run-local.sh
+```
 
-It will start a server at `localhost:9010`.
-If you run the script again, the server will be stopped and then restarted with the latest `extender.jar`
+The Extender server is now available on port `http://localhost:9000`
 
-### Debug
 
-#### Docker container
+### Stop the server
+You can stop the server by pressing `Ctrl-C` from the terminal prompt where it was started.
 
-* When the container is running, then run `./server/scripts/debug-local.sh`. It connects to the container using the `extender` user, and executes bash.
+---
 
-* In detail: [Debugging](./README_DEBUGGING.md)
+## Running as a standalone server on macOS
+The stand alone server is currently used on a machine runing macOS. The server is used to build darwin targets (macOS+iOS) using the Apple tools (XCode+Apple Clang)
 
-## Client
-
-There is a client part of this code which is used in Bob.jar.
-
-  1. Build the client
-
-    $ cd client
-    $ ../gradlew build
+### Run
+To run the stand alone server locally, you need to give it access to `/usr/local/extender`:
 
-  1. Copy the client to Bob
+```
+$ sudo mkdir /usr/local/extender
+$ sudo chown -R mawe:staff /usr/local/extender
+```
 
-    $ cp -v ./build/libs/extender-client-0.0.1.jar <defold>/com.dynamo.cr/com.dynamo.cr.common/ext/extender-client-0.0.1.jar
+Now the current user has access to the folder and can start the service.
 
+        $ TARGET_DIR=/path/to/localextender ./server/scripts/run-standalone-local.sh
```

Comment:
What are the prerequisites here in terms of package availability?

Reply:
Good point! It should be what we have in the Dockerfile right?

ENV XCODE_13_VERSION        13.2.1
ENV MACOS_12_VERSION        12.1
ENV IOS_15_VERSION          15.2
ENV XCODE_13_CLANG_VERSION  13.0.0
ENV SWIFT_5_5_VERSION       5.5